### ISSUE TITLE
Cannot uninstall Windows Terminal

### ISSUE THREAD

**Issue by DanHalford** [microsoft/terminal#11128]
On our corporate devices, the Microsoft Store is blocked. Windows Terminal was installed by downloading the MSIX package and installing manually. when trying to uninstall it:

- Windows Settings -> Apps -> Windows Terminal - Uninstall button is greyed out.
- From within an elevated Powershell session: `Remove-AppxPackage -Package Microsoft.WindowsTerminal`
```
Remove-AppxPackage : Deployment failed with HRESULT: 0x80073CFA, Removal failed. Please contact your software vendor.
(Exception from HRESULT: 0x80073CFA)
NOTE: For additional information, look for [ActivityId] 5773b13b-a048-0000-dcb3-7a5748a0d701 in the Event Log or use
the command line Get-AppPackageLog -ActivityID 5773b13b-a048-0000-dcb3-7a5748a0d701
At line:1 char:1
+ Remove-AppxPackage -Package Microsoft.WindowsTerminal
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (Microsoft.WindowsTerminal:String) [Remove-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.RemoveAppxPackageCommand
```
`Get-AppPackageLog -ActivityID 5773b13b-a048-0000-dcb3-7a5748a0d701`
```
Time                      ID           Message
----                      --           -------
3/09/2021 3:26:10 PM      603          Started deployment Remove operation on a package with main parameter
                                       Microsoft.WindowsTerminal and Options 0 and 0. See
                                       http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app
                                       deployment issues.
```

**Comment by zadjii-msft** [comment]
Weird. Are you absolutely sure there aren't any Terminal instances still running? Sometimes the app platform gets confused and leaves a handle open somewhere, or something like that. At the risk of sounding totally inane, does restarting the PC fix this by any chance?

If not then we'll have to follow up with the MSIX team

**Comment by Sanluli36li** [comment]
I cannot set the Terminal as the default console application. Is there a way to fix it?

![屏幕截图 2021-09-08 185928](https://user-images.githubusercontent.com/5201743/132498019-4dfdf85f-21a9-41fe-bb67-08b7a390822b.png)
![屏幕截图 2021-09-08 190028](https://user-images.githubusercontent.com/5201743/132498025-8aebc21d-c245-4d59-9231-805541ddc65f.png)

**Comment by lanfeust69** [comment]
Exactly the same thing here.

*Of course*, there is no instance running (even from an "offline" location after the workaround mentioned in #6010, which is the reason of the uninstall in the first place, as the install simply doesn't work at all), and the PC has been rebooted several times.

**Comment by lanfeust69** [comment]
New point of interest : in my case, it turned out that it was much worse than just not being able to uninstall.

I managed to track down a serious issue I had to this failed install : there seems to be an integration with the Explorer that tries to instantiate a COM component, and this eventually fails with a "network location cannot be reached" error (seen in windbg).

This was not much of a problem when directly on our corporate network, but when remoting with the VPN in the way, the error would happen only after a 20 seconds timeout. So the net effect was that double or right clicking any folder within the Explorer (including right-clicking on the background, which is basically the Desktop folder) resulted in a 20 seconds freeze. I removed a bunch of registry keys related to "WindowsTerminal" and associated ClassIds, and the problem disappeared.
